interfaces/vrep/DQ_VrepInterface.m

```diff
@@ -280,19 +322,34 @@ function stop_simulation(obj)
             t = DQ([0,double(object_position)]);
         end
         
-        %% Set Object Translation
-        function set_object_translation(obj,handle,t,relative_to_handle,opmode)
-            %% Set the translation of an object in V-REP
-            %%  >> t = DQ.i*0.01;
-            %%  >> vi.set_object_translation('DefaultCamera',t);
-            
+        
+        function set_object_translation(obj,objectname,translation,reference_frame,opmode)
+            % This method sets the translation of an object in the server.
+            % Usage:
+            %     set_object_translation(objectname,translation,reference_frame,opmode);  
+            %          objectname:  The object name.
+            %          translation: The desired translation. 
+            %          (optional) reference_frame:  Indicates the name of 
+            %                       the relative reference frame in which 
+            %                       the desired translation is expressed. 
+            %                       If not specified, the absolute frame is used.
+            %          (optional) opmode:The operation mode.
+            % Example:
+            %      t = DQ.i*0.01;
+            %      set_object_translation('DefaultCamera', t);
```

Comment:
If examples are provided for some parameters, then examples should be provided for all use cases. See the help function for `DQ.plot` for reference.